plugins/amazonq/chat/jetbrains-community/resources/META-INF/plugin-chat.xml

```diff
@@ -34,7 +34,9 @@
             <reference id="aws.toolkit.open.q.window"/>
         </group>
         <!-- TODO: q.openchat will eventually be in amazonq, aws.toolkit.q.sign.in will eventually be in core. -->
-        <action id="q.openchat" class="software.aws.toolkits.jetbrains.services.amazonq.QOpenPanelAction"/>
+        <action id="q.openchat" class="software.aws.toolkits.jetbrains.services.amazonq.QOpenPanelAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="meta alt Q" />
```

Comment:
This closes the IntelliJ IDE on mac. I tried to locally set this keybind as well to see if it would override the behavior and it does not. Is there another keybind to use instead?

Reply:
my mistake, yes you're right this keybind does not interfere. I mistook what meta/alt were binding to on Mac.